src/main.c

```diff
@@ -703,7 +650,11 @@ print_choices(int selection)
 	if ((line = calloc(sizeof(*line), line_length)) == NULL)
 		err(1, "calloc");
 
-	SLIST_FOREACH(choice, choices, choices) {
+	for (i = 0;
+	     i < (ssize_t)choices.nmemb
+	     && i < lines
```

Comment:
I think there might be an off by one error here. Running pick with more choices than would fit on screen, the selection is invisible. I think this is because the last choice printed to the terminal makes it scroll and then the first choice is overwritten by the query line.

Changing this to `&& i < lines + 1` locally fixes the issue for me.